File: output/CodeLlama/avatar/Python/Java/codeforces_31_A.java
import java.util.*;

public class codeforces_31_A {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int y = sc.nextInt();
        int[] arr = new int[y];
        for (int i = 0; i < y; i++) {
            arr[i] = sc.nextInt();
        }
        for (int i = 0; i < y; i++) {
            for (int j = 0; j < y; j++) {
                if (i == j) {
                    continue;
                }
                for (int k = 0; k < y; k++) {
                    if (arr[k] != arr[j] && arr[k] != arr[i] && arr[k] == arr[i] + arr[j]) {
                        System.out.println(k + 1 + " " + j + 1 + " " + i + 1);
                        return;
                    }
                }
            }
        }
        System.out.println(-1);
    }
}

File: output/CodeGen/avatar/Java/Python/codeforces_31_A.py
def main(args):
    sc = Scanner(System.in)
    n = sc.nextInt()
    a = [sc.nextInt() for _ in range(n)]
    f = 0
    for k in range(n):
        t = a[k]
        for i in range(n):
            for j in range(n - 1):
                if i!= j and t == (a[i] + a[j]):
                    f = 1
                    break
                if f == 1:
                    break
            if f == 1:
                break
        if f == 1:
            break
    if f == 1:
        print(k + 1 + " " + (j + 1) + " " + (i + 1))
    else:
        print("-1")

# Input:
# 5
# 1 2 3 4 5
# Output:
# 2 4 1